src/components/LoginPage/LoginPageComponent.js

```diff
@@ -1,67 +1,159 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import slide1 from 'components/LoginPage/images/slide-1.png';
-import slide2 from 'components/LoginPage/images/slide-2.png';
-import slide3 from 'components/LoginPage/images/slide-3.png';
-import slide4 from 'components/LoginPage/images/slide-4.png';
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { respondTo } from "_respondTo";
+import slide1 from "components/LoginPage/images/slide-1.svg";
+import slide2 from "components/LoginPage/images/slide-2.svg";
+import slide3description from "components/LoginPage/images/slide-3-description.svg";
+import slide4 from "components/LoginPage/images/slide-4.png";
+
+console.log("slide1", slide1);
+
+const OuterWrapper = styled.div`
+  background-color: #131c3e;
+`;
+
+const MainHeader = styled.h1`
+  display: block;
+  font-family: Roboto;
+  font-weight: 100;
+  font-size: 1.5rem;
+  line-height: 42px;
+  text-align: center;
+  color: #ffffff;
+  padding-top: 40px;
+  padding-bottom: 50px;
+
+  b {
+    color: #96ffff;
+  }
+
+  ${respondTo.xs`
+    font-size: 2rem;
+  `}
+  ${respondTo.sm`
+    font-size: 3rem;
+  `}
+  ${respondTo.xl`
+    font-size: 4rem;
+  `}
+`;
+
+const LoginButton = styled.a`
+  width: 250px;
+  display: block;
+  marginleft: 20px;
+  float: right;
+  margintop: 15px;
+`;
+
+const ImgSlide = styled.img`
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 370px;
+  ${respondTo.xs`
+    width: 450px;
+  `}
+  ${respondTo.sm`
+    width: 630px;
+  `}
+  ${respondTo.xl`
+    width: 850px;
+  `}
+`;
+
+const MediumHeader = styled.h2`
```

Comment:
Maybe it would be easier to stick to html naming?
```
const H2 = styled.h2`...`
```
Or `Heading2` or `HeadingLevel2`.

Reply:
`MediumHeader` is a strange name.

Also we could name it `SlideHeading`.